integration/js/server.js

```diff
@@ -40,7 +40,7 @@ createMeeting = async (createMeetingUrl) => {
       const data = await response.data;
       meetingMap[createMeetingUrl] = "created";
       console.log("Meeting created");
-      console.log(`Chime meeting id : ${data.JoinInfo.Meeting.MeetingId}`);
+      console.log(`Chime meeting id : ${data.JoinInfo.Meeting.Meeting.MeetingId}`);
```

Comment:
Why did we change this?

Reply:
This is an existing bug. The joinInfo structure is Meeting.Meeting.MeetingId.